test/parallel/test-tls-securepair-fiftharg.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var common = require('../common');
```

Comment:
`const` is supported by Node.js, I would prefer if you changed your variables to `const` wherever the value is assigned only once.